common/djangoapps/track/shim.py

```diff
@@ -119,15 +121,40 @@ def __call__(self, event):
 
             del payload['module_id']
 
+        if 'context' not in event:
```

Comment:
was this check needed for a certain app version?  Or did you just add this as defensive coding?